test/PoolManager.t.sol

```diff
@@ -617,6 +617,29 @@ contract PoolManagerTest is Test, Deployers, GasSnapshot {
         manager.setProtocolFees(key);
     }
 
+    function test_setProtocolFee_failsWithInvalidProtocolFeeControllers() public {
```

Comment:
i feel like this test could be kind of perfect for forge fuzzing
if forge sets a `uint256` input you can just do a bunch of if statements to check its behaving like you would expect
[see this test ](https://github.com/Uniswap/v4-core/blob/6b8ec7c6dff9a2bef4ff408f886b4cc1737179ae/test/PoolManagerInitialize.t.sol#L54)for an example

Reply:
Discussed offline and decided to leave as is for now, and after the fee refactor add a fuzz test for protocol swap fee updating

Reply:
Are we tracking this in alice's PR? I think I would like to see some fuzz tests for setting the protocol fee and just on the unit tests around _isValidProtocolFee

lgtm though